### PR TITLE
[botkit] Update Twilio Testbot

### PR DESCRIPTION
**Pull request by LCarbonellRod**
### Description

Twilio Testbot ready for release.

### Changes made

- Remove some unnecessary classes

- Remove some unnecessary dependencies

- Update comments and classes

- Update README.md file

### CODE REVIEW

File: samples/csharp_dotnetcore/52. twilio-adapter/Controllers/BotController.cs [hunk 1]
```diff
@@ -29,7 +31,7 @@ public async Task PostAsync()
         {
             // Delegate the processing of the HTTP POST to the adapter.
             // The adapter will invoke the bot.
-            await _adapter.ProcessAsync(Request, Response, _bot);
+                await _adapter.ProcessAsync(Request, Response, _bot, new CancellationToken());
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/52. twilio-adapter/Controllers/BotController.cs]: I think this is wrongly indented. Please fix it.

File: samples/csharp_dotnetcore/52. twilio-adapter/DeploymentTemplates/template-with-new-rg.json [hunk 2]
```diff
@@ -1,183 +1,183 @@
-{
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/52. twilio-adapter/DeploymentTemplates/template-with-new-rg.json:1]: What was the change in this file? If it was the end of line, you shouldn't apply that change

File: samples/csharp_dotnetcore/52. twilio-adapter/DeploymentTemplates/template-with-preexisting-rg.json [hunk 3]
```diff
@@ -1,154 +1,154 @@
-{
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/52. twilio-adapter/DeploymentTemplates/template-with-preexisting-rg.json:1]: Same for this file.

File: samples/csharp_dotnetcore/52. twilio-adapter/Properties/launchSettings.json [hunk 4]
```diff
@@ -1,27 +1,27 @@
-{
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/52. twilio-adapter/Properties/launchSettings.json:1]: Same here

File: samples/csharp_dotnetcore/52. twilio-adapter/README.md [hunk 5]
```diff
@@ -1,88 +1,93 @@
...
+  # determine dotnet version
+  dotnet --version
+  ```
+
+## To try this sample
+
+1 - Clone the repository
+
+    ```bash
+    git clone https://github.com/Microsoft/botbuilder-dotnet.git
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/52. twilio-adapter/README.md]: Please fix the repository's name

File: samples/csharp_dotnetcore/52. twilio-adapter/Startup.cs [hunk 6]
```diff
@@ -7,38 +7,23 @@
...
 
-            // Create the credential provider to be used with the Bot Framework Adapter.
-            services.AddSingleton<ICredentialProvider, ConfigurationCredentialProvider>();
-
-            // Create the options for the Twilio Adapter
-            services.AddSingleton<ITwilioAdapterOptions, ConfigurationTwilioAdapterOptions>();
-
-            // Create the Twilio Adapter.
-            services.AddSingleton(_adapter);
+            // Create the Bot Framework Webex Adapter.
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/52. twilio-adapter/Startup.cs]: Fix `Webex` in this comment

File: samples/csharp_dotnetcore/52. twilio-adapter/appsettings.Development.json [hunk 7]
```diff
@@ -1,9 +1,9 @@
-{
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/52. twilio-adapter/appsettings.Development.json:1]: Again the end of line.

File: samples/csharp_dotnetcore/52. twilio-adapter/appsettings.json [hunk 8]
```diff
@@ -1,8 +1,8 @@
...
-  "MicrosoftAppPassword": "",
-  "TwilioNumber": "",
-  "AccountSid": "",
-  "AuthToken": "",
-  "ValidationUrl": ""
-}
\ No newline at end of file
+{
+  "MicrosoftAppId": "",
+  "MicrosoftAppPassword": "",
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/52. twilio-adapter/appsettings.json]: Please remove these tokens.

File: samples/csharp_dotnetcore/52. twilio-adapter/README.md [hunk 9]
```diff
@@ -1,88 +1,93 @@
...
+  ```bash
+  # run the bot
+  dotnet run
+  ```
+
+  B) Or from Visual Studio
+
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `test/Test Applications/Microsoft.Bot.Builder.Twilio.Sample` folder
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/52. twilio-adapter/README.md]: Fix the path here.

File: samples/csharp_dotnetcore/52. twilio-adapter/README.md [hunk 10]
```diff
@@ -1,88 +1,93 @@
...
+
+## To try this sample
+
+1 - Clone the repository
+
+    ```bash
+    git clone https://github.com/Microsoft/botbuilder-dotnet.git
+    ```
+
+2 - In a terminal, navigate to `test/Test Applications/Microsoft.Bot.Builder.Twilio.Sample`
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/52. twilio-adapter/README.md]: Please fix the path here.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

File: samples/csharp_dotnetcore/52. twilio-adapter/DeploymentTemplates/template-with-new-rg.json (hunk 2 above)
Comment (ceciliaavila) [reply samples/csharp_dotnetcore/52. twilio-adapter/DeploymentTemplates/template-with-new-rg.json:1]: Let's leave this for now.

**Review by ceciliaavila — APPROVED** [review]
LGTM

**Review by matiasroldan6 — APPROVED** [review]

**Pull request merged**